doc/spelling_wordlist.txt

```diff
@@ -0,0 +1,118 @@
+addon
+addons
+api
+auth
+autobuild
+backend
+backends
+banktransfer
+boolean
+booleans
+cancelled
+casted
+checkbox
+checkin
+checkins
```

Comment:
We could discuss (not in the scope of this PR) if we write ``checkin`` or ``check-in`` consistently, I think it is inconsistent right now. Do you happen to have an opinion on this?

Reply:
As of now, I don't. I'll have a look into how inconsistenly this is currently used, and report back, maybe also with an opinion ;)